src/app.css

```diff
@@ -0,0 +1,25 @@
+html, body{
```

Comment:
I wonder whether this will collide with any of David's work on the new registration page. Can you pull from his branch (`newregistrationpage`, hopefully soon-to-be-merged), and see if there are any collisions?